config/datanode.example.toml

```diff
@@ -10,6 +10,8 @@ rpc_addr = "127.0.0.1:3001"
 rpc_hostname = "127.0.0.1"
 # The number of gRPC server worker threads, 8 by default.
 rpc_runtime_size = 8
+# Whether to enable greptimedb telemetry, true by default.
+enable_telemetry = true
```

Comment:
I think `enable_telemetry` shouldn't been placed here. I'm aware it's passed by standalone mode, but in that case, it should be treated like an internal config.

Reply:
Good catch, the user should not set this parameter for datanode.